index.html

```diff
@@ -21,11 +21,10 @@
 	<div class="logo_index">
 		<div class="nav">
 			<ul>
-				<li a href="index.html"><img src="images/home.png" height="40" alt="logo"></a>
+				<li><a href="index.html"><img src="images/home.png" height="40" alt="logo"></a>
 				<ul>
-					<li class="four"><a href="#grid3">About Us</a></li>
-					<li class="five"><a href="#grid4">What to Bring</a></li> 
-						<hr />
+					<li><a href="#grid3" class="underline">About Us</a></li>

```

Comment:
I improved the position of the page links to show the entire section downwards on click not the header downwards